app/assets/javascripts/govuk_publishing_components/lib/cookie-functions.js

```diff
@@ -201,13 +201,18 @@
   }
 
   window.GOVUK.deleteCookie = function (cookie) {
```

Comment:
This all looks fine but I'm trying to get my head around the logic of it, bear with me.

Originally, to delete a cookie, we would call `window.GOVUK.cookie` with the cookie name and null, which I think clears the content of the cookie. Then an `if` statement would check if that cookie existed (which presumably always was the case) and then within it set the expiry date for that cookie. Now, we have the `if` first, followed by the call to `window.GOVUK.cookie`.

I guess I have two questions - firstly, is the `if` statement really needed, and secondly what difference does changing the order of these two things make? (if the answer to the first is 'no' then I think the second question is redundant).

Reply:
@andysellick Good point - I don't think the `if statement` is needed, so have removed and put the order back.